lista/40/renan.js

```diff
@@ -0,0 +1,14 @@
+// 40) Crie um aplicativo que mostre na tela a seguinte contagem:
```

Comment:
Bom exercício.
Uma alternativa para simplificar o código é colocar o print do "acabou" após o final do corpo do loop, assim poupa um if. 
Fazendo com um for também fica relativamente simples

```
for (let n = 0; n <= 18; n *= 3) console.log(n)
console.log("acabou!")
```